### ISSUE TITLE
Unable to interact with content of main menu using VO on iOS

### ISSUE THREAD

**Issue by joshblack** [carbon-design-system/carbon-website#1130]
This could easily be an upstream UI Shell issue, if so let us know and we can move this over 👀

Noticed this while on the website in Safari, it seems like we're unable to access the content inside of the menu.

Steps to reproduce:

[cannot-navigate-in-menu.mov.zip](https://github.com/carbon-design-system/carbon-website/files/4540209/cannot-navigate-in-menu.mov.zip)
